frontend/src/v5/ui/components/dashboard/dashboardList/dashboardListCollapse/dashboardListCollapse.component.tsx

```diff
@@ -44,7 +44,7 @@ export const DashboardListCollapse = ({
 	const [isExpanded, setIsExpanded] = useState(defaultExpanded);
 
 	return (
-		<Container className={className} isLoading={isLoading}>
+		<div className={className} >
```

Comment:
Getting rid of the `<Container />` here messes up the padding in the tickets list. (Notice the column headers for instance)
![image](https://github.com/3drepo/3drepo.io/assets/62481250/6b6a3bff-dcaa-4816-8fd9-b8b568747cc4)

You can see in `ticketsTableContent.styles` that <Container /> is styled